2996_reflection/reflection.md

```diff
@@ -937,19 +939,19 @@ struct Clap {
           opts.[:om:] = *cur.initializer;
           continue;
         } else {
-          std::print(stderr, "Missing required option {}\n", name_of(sm));
+          std::print(stderr, "Missing required option {}\n", display_string_of(sm));
```

Comment:
This one should be fine as `identifier_of` — `sm` is a reflection of a non-static data member. I guess it's the same either way. 

Ditto throughout.

Reply:
I would prefer to keep as-is: It happens that `identifier_of(sm)` is fine for non-static data members, but if we were instead outputting a type (as we do below), then it's _not_ necessarily safe - e.g., the type may be an anonymous struct. Rather than draw attention to that distinction, I'm inclined to just blanket-use `display_string_of` everywhere in the example(s) that we're formatting a reflection into text. Let me know if you have a strong preference to use `identifier_of` though, and I can change it.